### ISSUE TITLE
lpd8 doesnt work at all

### ISSUE THREAD

**Issue by kubazrb** [stylemistake/bitwig-lpd8#3]
Hi, 
I use KxStudio (Linux), and when I open bitwig I can see statement that LPD8 is pluged but when i hit a pad or move knob nothing happens. I've tried to add controller manually and automatically  but the result was same. What can be wrong?

**Comment by stylemistake** [comment]
That's a known issue, but I don't have time to fix it. The problem is that LPD8 presets are not transferred using SysEx messages from my controller script. It should be fixable, it worked in 1.0 versions. Pull requests are welcome.
P.S. Unless you have some other than a preset problem.

**Comment by swilliams-dev** [comment]
Can we get a dump of your LPD8 @stylemistake? That may be a good workaround, and once mine works I may be able to figure out a shim / or PR.

**Comment by stylemistake** [comment]
@swilliams-dev no, but there are some clues on how this preset was structured from the code

https://github.com/stylemistake/bitwig-lpd8/blob/master/lib/lpd8.js#L46-L56

So, from what I can remember

- CC messages from knobs were in the range of 32-63 (8 knob ids * 4 presets)
- CC messages from pads were in the range of 64-95 (8 pad ids * 4 presets)
- MIDI notes from pads were in the range of 36-67 (C2-G4) (8 pad notes * 4 presets)

I don't remember if midi channels really mattered. This should be enough info to reconstruct the preset.

**Comment by stylemistake** [comment]
@swilliams-dev share a preset perhaps.
